deptofdefense/move.mil
# Tools & Resources page

## Checklist

I have…


## Summary of Changes

This pull request…

Adds the tools and resources page. Issue #108

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
2. git checkout tools,
3. set up development dependencies according to CONTRIBUTING.md,
4. run bin/rails server, and
5. load up http://localhost:3000 in your Web browser of choice, click links on tools & resources page

## Screenshots

Full screen 1:
<img width="1147" alt="screen shot 2017-11-01 at 9 56 25 am" src="https://user-images.githubusercontent.com/29409348/32278311-a5c12238-beeb-11e7-850c-f30f55af4c97.png">

Full screen 2:
<img width="1093" alt="screen shot 2017-11-01 at 9 56 33 am" src="https://user-images.githubusercontent.com/29409348/32278312-aa4ddf62-beeb-11e7-9e07-e8fa25c94501.png">

Narrow 1:
<img width="396" alt="screen shot 2017-11-01 at 9 56 02 am" src="https://user-images.githubusercontent.com/29409348/32278333-bce093d6-beeb-11e7-8041-5b8ba55005a1.png">

Narrow 2:
<img width="393" alt="screen shot 2017-11-01 at 9 56 11 am" src="https://user-images.githubusercontent.com/29409348/32278338-c03635c2-beeb-11e7-9a55-d8f63dc1079d.png">
